addons/account/models/account_tax_report.py

```diff
@@ -112,14 +112,22 @@ class AccountTaxReportLine(models.Model):
     formula = fields.Char(string="Formula", help="Python expression used to compute the value of a total line. This field is mutually exclusive with tag_name, setting it turns the line to a total line. Tax report line codes can be used as variables in this expression to refer to the balance of the corresponding lines in the report. A formula cannot refer to another line using a formula.")
 
     # fields used to carry over amounts between periods
-    carry_over_condition_method = fields.Char(string="Condition for carry over",
-                                              help="The name of the method used to check if a carry over is needed.")
-    carry_over_destination_line_id = fields.Many2one(string="Carry over line",
-                                                     comodel_name="account.tax.report.line",
-                                                     help="The line to which the value of this line will be carried over to if needed. If left empty the line will carry over to itself.")
-    carry_over_analytic_account_id = fields.Many2one(string="Analytic account",
-                                                     comodel_name="account.analytic.account",
-                                                     help="The analytic account used to carry over amounts between periods. If none is selected, a new acount will be created when necessary.")
+    # Turn this into a select maybe?
+    carry_over_condition_method = fields.Char(
+        string="Condition for carry over",
+        help="The name of the method used to check if a carry over is needed."
```

Comment:
I don't see the utility of this

Reply:
It was made like that because different localisations may need different conditions for carrying over. 
But also, it would not shock me that in some cases, we may need to have two lines in the same report that needs two different conditions for carrying them over.
I may be wrong on the last part, I will need to discuss this more with TSB and/or AVW to see if this is really useful.